### ISSUE TITLE
Importing xk6 kafka breaks net/grpc

### ISSUE THREAD

**Issue by SameeranB** [mostafa/xk6-kafka#174]
For some reason `import {Connection, Writer, SchemaRegistry, SCHEMA_TYPE_JSON} from 'k6/x/kafka';` makes it such that the response from a gRPC client is empty: `const response = client.invoke('auror.Auror/GenerateToken', {secret: "strongpass"});`

I have tried removing each exported function from the import statement. But it would seem that just importing `k6/x/kafka` itself breaks grpc.

This is the code:
```javascript
import {Connection, Writer, SchemaRegistry, SCHEMA_TYPE_JSON} from 'k6/x/kafka';
import grpc from "k6/net/grpc";

export default function() {

    client.connect('localhost:3001', {plaintext: true});

    const response = client.invoke('auror.Auror/GenerateToken', {secret: "strongpass"});
    check(response, {
        'status is OK': (r) => r && r.status === grpc.StatusOK,
    });
    console.log(JSON.stringify(response.message));
}
```

This is the output with kafka imported:
```bash
INFO[0000] undefined                                     source=console
INFO[0000] undefined                                     source=console
INFO[0000] undefined                                     source=console
INFO[0000] undefined                                     source=console
INFO[0000] undefined                                     source=console
INFO[0000] undefined                                     source=console
...
```

This is the output without the import statement:
```bash
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console
INFO[0000] {"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IiIsImV4cCI6MTY2ODE2MzY2N30.nW9FqhxuG2gFTkkZiq_6ktIFE4z_1ggB8DOVTSfm92k"}  source=console

```

I'm sure I'm doing something wrong... any ideas?

**Comment by mostafa** [comment]
Hey @SameeranB,

I can confirm that importing the extension causes the GRPC to fail, however I currently don't know the actual cause and I need to investigate it.

**Comment by mostafa** [comment]
@SameeranB 

I released a new version, [v0.15.1](https://github.com/mostafa/xk6-kafka/releases/tag/v0.15.1), that fixes this bug.
